tsfc/fiatinterface.py

```diff
@@ -178,6 +173,15 @@ def _(element, vector_is_mixed):
         # Real element is just DG0
         cell = element.cell()
         return create_element(ufl.FiniteElement("DG", cell, 0), vector_is_mixed)
+    if element.family() == "Quadrature":
+        # Sneaky import from FFC
+        from ffc.quadratureelement import QuadratureElement
+        ffc_element = QuadratureElement(element)
+
+        def tabulate(self, order, points, entity):
+            return QuadratureElement.tabulate(self, order, points)
+        ffc_element.tabulate = types.MethodType(tabulate, ffc_element)
+        return ffc_element
```

Comment:
> The FiniteElement/CiarletElement split somewhat mitigates this problem.

I don't think that this is fully correct. As of I remember, some FFC elements implement only tabulate and do not bother with, for instance, expansion sets, dmats, etc. But I agree, it would be good to fix this.